token-lending/program/tests/helpers/mod.rs

```diff
@@ -521,6 +525,7 @@ impl TestLendingMarket {
             authority: lending_market_authority,
             quote_currency: QUOTE_CURRENCY,
             oracle_program_id,
```

Comment:
nit: add the key here? pyth_oracle_program_id or whatever it is
just so it matches